app/femr/ui/controllers/SessionsController.java

```diff
@@ -77,6 +84,23 @@ public Result createPost() {
             if (user.getPasswordReset() == true){
                 return editPasswordGet(user);
             }
+
+            // put internet check here
+            if (InternetCheck.NetIsAvailable()) {
```

Comment:
could be a little more concise. only difference in code below is value passed to updateInternetStatus()